### PR TITLE
feat(spanner): update samples for optimizer statistics package

### PR DESCRIPTION
**Pull request by hengfengli**

### CODE REVIEW

**Comment by hengfengli** [comment]
Wait for the release 2.7.0 and need to update the `google-cloud-spanner` to `2.7.0`.

**Review by skuruppu — APPROVED** [review]

**Comment by hengfengli** [comment]
@JustinBeckwith Can I get an approval for this PR? Thanks!

**Review by dazuma — APPROVED** [review]

**Pull request merged**
